account_invoice_reorder_lines/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+      :alt: License: AGPL-3
+
+==================================
+Invoice lines with sequence number
+==================================
+
+This module was written to provide a new field on the invoice form, allowing to manage the invoice lines
```

Comment:
I mean this text. The field is already there. You are only providing a different order.